src/large_pages/node_large_page.cc

```diff
@@ -251,6 +263,40 @@ struct text_region FindNodeTextRegion() {
       size = 0;
     }
   }
+#elif defined(__sun)
+  auto size = 1 << 20;
+  auto fd = open("/proc/self/map", O_RDONLY);
+
+  if (fd != -1) {
+    ssize_t r;
+    prmap_t* cur, * map;
```

Comment:
Could you also run clang-format on the changes, please?
`make format-cpp-build` - to build the formatter and then
`CLANG_FORMAT_START=master make format-cpp` to format changes on your branch.
OR
``` CLANG_FORMAT_START=`git rev-parse HEAD~1` make format-cpp ``` to format changes in the last commit.

Reply:
sorry for late reply takes age to build in the VM :-) yes sure fair point will fix style.